Quadriga/src/main/webapp/WEB-INF/views/status_transform.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8"
+    pageEncoding="UTF-8"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+<META HTTP-EQUIV="refresh" CONTENT="5">
+<title>Insert title here</title>
+</head>
+<body>
+
+<h1> Your page is downloading ... </h1>
```

Comment:
there should be a heading ("Transformation Successful") and a paragraph that says "The resulting graph of the transformation is being downloaded."